src/ui-kit/form-controls/upload-v2/upload-v2.template.html

```diff
@@ -1,3 +1,16 @@
+<sam-modal *ngIf="uploadFileActionModalConfig" #actionModal
+           [showClose]="true"
+           [closeOnOutsideClick]="true"
+           [closeOnEscape]="true"
+           [type]="'warning'"
+           [title]="uploadFileActionModalConfig.title"
+           [submitButtonLabel]="'Ok'"
+           [cancelButtonLabel]="'Cancel'"
+           (onSubmit)="onActionModalSubmit($event)"
+           (close)="onUploadModalCloseChange($event)">
+  <ng-content select=".upload-v2-action-modal"></ng-content>
```

Comment:
the `upload-v2` is alittle redundant, can you use `.action-modal-content`?

Reply:
Sure